LeaderboardBackend/Models/Entities/Leaderboard.cs

```diff
@@ -70,5 +76,9 @@ public void Configure(EntityTypeBuilder<Leaderboard> builder)
 
         builder.Property(l => l.Info)
             .HasDefaultValue("");
+
+        builder.HasIndex(lb => lb.Name)
+            .HasMethod("GIN")
+            .IsTsVectorExpressionIndex("english");
```

Comment:
Do we know if the index actually gets used? I'm testing it on my end right now, against a table with 100 rows, and my `explain` output shows Postgres isn't using the index at all (SQL's from console output):
<img width="946" alt="image" src="https://github.com/user-attachments/assets/ce09f7a8-deff-470a-b331-dda0ab4bbce7" />